src/applications/pensions/config/form.js

```diff
@@ -1449,6 +1449,12 @@ const formConfig = {
               'ui:title':
                 'Do you have any medical, legal or other unreimbursed expenses?',
               'ui:widget': 'yesNo',
+              'ui:options': {
+                // HACK: Forcing wrapper to be a <div> instead of <dl>
+                // in order to avoid breaking accessibility.
+                customTitle: ' ',
```

Comment:
This hacky pattern seems to be used in some other forms (found by searching `customTitle`).

[The logic for this can be found in this code](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/review/ObjectField.jsx#L125).